### PR TITLE
Bump Ray Version to 1.3.0

### PR DESCRIPTION
**Pull request by kira-lin**
Ray 1.3.0 Jar is ready.

### CODE REVIEW

**Comment by kira-lin** [comment]
It seems like Ray Tune in 1.3.0 has a [bug](https://github.com/ray-project/ray/issues/15504). In this PR I'll change the CI to bypass it.

File: .github/workflows/raydp.yml [hunk 1]
```diff
@@ -63,7 +63,13 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install wheel
-          pip install pytest koalas tensorflow torch tabulate ray[tune] xgboost_ray grpcio-tools
+          if [ "$(uname -s)" == "Linux" ]
+          then
+            pip install torch==1.8.1+cpu -f https://download.pytorch.org/whl/torch_stable.html
```
Comment (ConeyLiu) [inline .github/workflows/raydp.yml:68]: Why we need this?

File: core/pom.xml [hunk 2]
```diff
@@ -21,7 +21,7 @@
     <fasterxml.jackson.version>2.12.2</fasterxml.jackson.version>
     <jersey.version>2.31</jersey.version>
     <scala.binary.version>2.12</scala.binary.version>
-    <ray.version>1.2.0</ray.version>
+    <ray.version>1.3.0</ray.version>
```
Comment (ConeyLiu) [inline core/pom.xml:24]: Does the package has published? I can't find it: https://mvnrepository.com/artifact/io.ray/ray-api

**Comment by ConeyLiu** [comment]
Should we also upgrade the python ray dependency?

File: .github/workflows/raydp.yml (hunk 1 above)
Comment (kira-lin) [reply .github/workflows/raydp.yml:68]: pip install torch in Linux will download ~800M, while torch-cpu is only ~100M. And we don't need gpu anyway.

File: core/pom.xml (hunk 2 above)
Comment (kira-lin) [reply core/pom.xml:24]: It's published. You can check it now. I saw it [here](https://github.com/ray-project/ray/issues/15496#issuecomment-827298371)

**Comment by kira-lin** [comment]
I have upgraded the required ray version. I think we can merge this PR first because it has the fix for CI. @ConeyLiu

**Comment by ConeyLiu** [comment]
thanks.

**Pull request merged**